website/src/reference/builder/member/getter.md

```diff
@@ -0,0 +1,87 @@
+# `overwritable` :microscope:
+
+**Applies to:** <Badge type="warning" text="struct fields"/> <Badge type="warning" text="function arguments"/> <Badge type="warning" text="method arguments"/>
+
+Allows getting a reference to an already set member.
+
+::: danger 🔬 **Experimental**
+
+This attribute is available under the cargo feature `experimental-getter`. Breaking changes may occur between **minor** releases but not between patch releases.
```

Comment:
Don't forget to add this cargo feature into `Cargo.toml` of `bon` and `bon-macros`, and add validation to the code that reads the config that makes sure that feature is enabled before one could use the `builder(getter)` attribute